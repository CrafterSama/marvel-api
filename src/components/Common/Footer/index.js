import React from 'react'
import { FooterContainer } from '../../../styles'
import { useStateValue } from '../../../context';

const Footer = () => {
  const [state] = useStateValue()
  return (
    <FooterContainer darkmode={state.darkMode}>
      <p>
        Made by <a href="https://craftersama.com" target="_blanc">Julmer Olivero</a>
      </p>
    </FooterContainer>
  );
}

export default Footer
