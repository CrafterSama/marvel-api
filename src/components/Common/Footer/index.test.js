import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJSON from 'enzyme-to-json';
import Footer from './index';

configure({ adapter: new Adapter() });

describe('Test for <Footer /> Component', () => {
  test('matches snapshot', () => {
    const wrapper = shallow(<Footer />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
