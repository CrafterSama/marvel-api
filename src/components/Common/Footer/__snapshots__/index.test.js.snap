// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for <Footer /> Component matches snapshot 1`] = `
<styles__FooterContainer
  darkmode=""
>
  <p>
    Made by 
    <a
      href="https://craftersama.com"
      target="_blanc"
    >
      Julmer Olivero
    </a>
  </p>
</styles__FooterContainer>
`;
